### PR TITLE
:lipstick: #971 - The styling of the select widget with multiple valu…

### PR DESCRIPTION
**Pull request by svenvandescheur**
…es is off.

### CODE REVIEW

**Comment by svenvandescheur** [comment]
Schermafbeelding 2021-11-26 om 12 59 30

File: src/scss/components/_select.scss [hunk 1]
```diff
@@ -21,31 +23,83 @@
 // Styling for the choicejs widget for the select component
 // Overwriting styles found here: https://github.com/Choices-js/Choices/blob/master/src/styles/choices.scss
 .#{prefix(form-control--select)} {
+  $control-height: $grid-row-height;
+  $list-hpadding: $grid-margin-2;
+  $list-vpadding: $grid-margin-1;
+  $list-vpaddings:  $list-vpadding * 2;
+  $item-margin: $grid-margin-0/2;
```
Comment (sergei-maertens) [inline src/scss/components/_select.scss:30]:
```suggestion
  $item-margin: math.div($grid-margin-0, 2);
```

File: src/scss/components/_select.scss [hunk 2]
```diff
@@ -21,31 +23,83 @@
...
+  // Style as a regular input.
   .choices {
     @include body;
     @include border(all, $size: 1px);
-    @include rows(1);
-    appearance: none;
+    @include rows(1, min-height);
+    align-items: center;
     border-radius: 0;
+    display: flex;
```
Comment (sergei-maertens) [inline src/scss/components/_select.scss:45]: can you please keep `display: flex` and `align-items` grouped together, as they both apply to the flexbox layout

now it's harder to read how the layout is defined.

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

**Review by sergei-maertens — APPROVED** [review]

**Comment by sergei-maertens** [comment]
Some more tweaks

image

**Pull request merged**